### DISCUSSION TITLE
Gitlab self-hosted and Gitlab.com packages

### DISCUSSION THREAD

**Discussion by joey-bolts** [renovatebot/renovate#19380] [Request Help]
Hi,

I am running renovate in gitlab.xxx.xx (self-hosted) this works very well and most of our packages come from github.
Now i encountered a project that also has a package in composer.json where the source is located on gitlab.com
Which in CI run says it does not have permissions. Is it possible use both gitlab instances? If so how would i configure that with access tokens/keys?

**Comment by rarkins** [comment]
You should be able to configure a `hostRules` entry with matchHost=gitlab.com
